src/main/webapp/app/pages/oncologyTherapiesPage/oncologyTherapiesPage.tsx

```diff
@@ -0,0 +1,679 @@
+    'The exact year of the drug’s first FDA-approval could not be determined due to absent or ambiguous data on FDA.gov website',
+  fdaUnapproved: (
+    <span>
+      The drug is no longer FDA-approved according to{' '}
+      <Linkout
+        link={
+          'www.fda.gov/drugs/resources-information-approved-drugs/withdrawn-cancer-accelerated-approvals'
+        }
+      >
+        fda.gov
+      </Linkout>
+      .
+    </span>
+  ),
+};
+
+const DefinitionTooltip: React.FunctionComponent<{
+  className?: string;
+  footnoteKey: keyof typeof footnotes;
+}> = props => {
+  return (
+    <InfoIcon
+      className={props.className ? props.className : 'ml-2'}
+      overlay={<span>{footnotes[props.footnoteKey]}</span>}
+    />
+  );
+};
+const OncologyTherapiesPage: React.FunctionComponent<{}> = props => {
+  const [showDefinition, setShowDefinition] = useState(false);
+
+  const [hasFilter, setHasFilter] = useState(false);
+  const [poOnly, setPoOnly] = useState(true);
+  const [targetedOnly, setTargetedOnly] = useState(false);
+  const [biomarkerSearch, setBiomarkerSearch] = useState('');
+  const [selectedTherapies, setSelectedTherapies] = useState<SelectOption[]>(
+    []
+  );
+  const [selectedAgents, setSelectedAgents] = useState<SelectOption[]>([]);
+  const [selectedDrugTargets, setSelectedTargets] = useState<SelectOption[]>(
+    []
+  );
+  const [classificationStats, setClassificationStats] = useState<{
+    [key: string]: number;
+  }>({});
+
+  const simplifyDrugTarget = (drugTarget: string) =>
+    drugTarget.replace(/\(.*\)/, '').trim();
+  const columns: SearchColumn<FdaApprovedOncologyTherapy>[] = [
+    {
+      accessor: 'year',
+      Header: <span>Year of drug’s first FDA-approval</span>,
+      Cell(tableProps: { original: FdaApprovedOncologyTherapy }) {
+        if (tableProps.original.year.endsWith('*')) {
+          return (
+            <span>
+              {tableProps.original.year.slice(
+                0,
+                tableProps.original.year.length - 1
+              )}
+              <DefinitionTooltip footnoteKey={'unclearYear'} />
+            </span>
+          );
+        } else {
+          return tableProps.original.year;
+        }
+      },
+    },
+    {
+      accessor: 'tx',
+      Header: <span>FDA-approved drug(s)</span>,
+      Cell(tableProps: { original: FdaApprovedOncologyTherapy }) {
+        if (tableProps.original.tx.endsWith('*')) {
+          return (
+            <span>
+              {tableProps.original.tx.slice(
+                0,
+                tableProps.original.tx.length - 1
+              )}
+              <DefinitionTooltip footnoteKey={'fdaUnapproved'} />
+            </span>
+          );
+        } else {
+          return tableProps.original.tx;
+        }
+      },
+    },
+    {
+      show: poOnly,
+      Header: (
+        <span>
+          FDA drug label listed biomarker(s)
+          <DefinitionTooltip footnoteKey={'biomarker'} />
+        </span>
```

Comment:
Should sorting in this column be disabled? Doesn't work very well right now since we also have bulleted items in the cell